<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1371</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_Main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1281</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cpu</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_Cpu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>791</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_memView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>791</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_Stack">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>421</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PE View</string>
     </attribute>
     <widget class="QTreeView" name="treeView_Modules">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>481</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1371</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuInject">
     <property name="title">
      <string>Inject</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action1"/>
     <addaction name="action2"/>
     <addaction name="action3"/>
     <addaction name="action4"/>
     <addaction name="action5"/>
     <addaction name="action6"/>
     <addaction name="action7"/>
    </widget>
    <addaction name="actionAttach_Process"/>
    <addaction name="menuInject"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAttach_Process">
   <property name="text">
    <string>Attach Process</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
